data_sources/wikipedia/cases/Snakefile

```diff
@@ -1,5 +1,9 @@
 WP_TS = 'country-level/{country_code}_ts.csv'
-COUNTRIES = ['USA']
+
+#COUNTRIES = ['KOR']
+f_in = open('country_case_wp_pages.csv', "r")
+COUNTRIES = [text.split(",")[0] for text in list(f_in)][1:]
+f_in.close()
```

Comment:
I think it's simpler to use pandas? 

```python
pd.read_csv(....)['country_code'].to_list()
```

Reply:
I wanted to avoid external library as much as I can, yet I agree with you.